Rakefile

```diff
@@ -8,7 +8,13 @@ require "ruby_lsp/check_docs"
 Rake::TestTask.new(:test) do |t|
   t.libs << "test"
   t.libs << "lib"
-  t.test_files = FileList["test/**/*_test.rb", "lib/ruby_indexer/test/**/*_test.rb"].exclude("test/fixtures/prism/**/*")
+  t.test_files = FileList["test/**/*_test.rb"].exclude("test/fixtures/prism/**/*")
+end
+
+Rake::TestTask.new(:indexer_test) do |t|
```

Comment:
Could we put this inside the `test` namespace so it's `rake test:indexer`? That would be closer to conventions elsewhere, e.g. Rails.

Reply:
Great idea!